schemas/property.py

```diff
@@ -26,6 +26,8 @@ def validates_property_manager_ids(self, value):
 
     @validates("name")
     def validates_uniqueness_of_name(self, value):
+        if "name" in self.context and value == self.context["name"]:
+            return value
         if PropertyModel.find_by_name(value):
             raise ValidationError("A property with this name already exists")
```

Comment:
I see that we're passing context "PUT" everywhere, but I don't actually see it used anywhere. Is that context info necessary?

I find this logic a bit confusing, although after some thought I realize that we are only returning early when the name has been passed in which only occurs on the PUT action. So it seems we can remove passing in the `PUT` context. I think if we do something like this it might be a bit more readable.
```suggestion
        if PropertyModel.find_by_name(value) and updating_name:
            raise ValidationError("A property with this name already exists")
```

and move the necessary log to a private method for `updating_name`

Reply:
I changed this so that it now uses a private method called `_assigning_name()`.  Since this validation should could return an error from a property name update OR the creation for a new property, I thought that name might be more appropriate.  Let me know what you think of this new implementation.